recipes/opencadd/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "opencadd" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/opencadd/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: d5558cd419c8d46bdc958064cb97f963d1ea793866414c025906ec15033512ed
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - pandas
+    - mdanalysis
+    - nglview
+    - bravado
+    - biopandas
+    - rdkit
+    - tqdm
+    - biotite
+    - biopython 1.77
```

I think we can be a little less strict here.

```suggestion
    - biopython <=1.77
```